web/src/app/modules/shared/components/presentation/datagrid/datagrid.component.html

```diff
@@ -1,4 +1,11 @@
-<h4 *ngIf="showTitle()">{{ title }}</h4>
+<div class="clr-row clr-align-items-center clr-justify-content-between" *ngIf="showTitle() || buttonGroup">
```

Comment:
Why did you choose this method over Clarity's provided clr-dg-action-bar?

Reply:
Example here https://clarity.design/documentation/datagrid/batch-action

Reply:
The short answer is that I wasn't aware of `clr-dg-action-bar`. The component itself [doesn't do much](https://github.com/vmware/clarity/blob/c77dee037dda21556dba30aa4eb742ea1832677b/packages/angular/projects/clr-angular/src/data/datagrid/datagrid-action-bar.ts), it's just a semantic concept.

Added, but keeping it outside of `clr-datagrid` to preserve the basic layout of the button to the side of the title. By default the buttons appear below the title, which is odd to my eyes, but that opinion is weakly held.